test/party_client.py

```diff
@@ -434,3 +434,31 @@ def get_pending_surveys_with_originator_party_id(self, party_id, expected_status
     def delete_pending_surveys_with_batch_no(self, batch_no, expected_status=202):
         response = self.client.delete(f"/party-api/v1/pending-surveys/{batch_no}", headers=self.auth_headers)
         self.assertStatus(response, expected_status)
+
+    def get_password_verification_token(self, party_id, expected_status=200):
```

Comment:
Is this used? I can't see where

Reply:
I had added these for completeness but they are now removed